### ISSUE TITLE
Grid: New Flex property in header causes layout issues in grids

### ISSUE THREAD

**Issue by IanMuirhead** [telerik/kendo-theme-default#653]
With the change in #652 it has introduced a bug when scrolling wide grids with `[scrollable]="'none'"`.

![screen shot 2017-07-28 at 12 02 54 pm](https://user-images.githubusercontent.com/884395/28732438-bb6df336-738c-11e7-9324-02a96cfe1b09.png)

@gyoshev

**Comment by gyoshev** [comment]
Thanks! I'll add this to the test cases and we'll strive to push a fix as soon as possible.

**Comment by gyoshev** [comment]
Actually, the described problem seems to be the regression from e6ac33596b3b47b3a4a4ad1b089efd3d8dc4ac7d that was fixed with #652 . The issue should already be resolved in `v2.39.0-dev.201707281346` -- if not, can you please provide the grid configuration?

**Comment by IanMuirhead** [comment]
@gyoshev I opened the default grid Plunk from the grid overview on the components section of the kendo website and included the following snippet (not sure how to link to the develop all css from Github).

` 
[scrollable]="'none'"
`

```
.k-grid-header, .k-grid-footer {
    -webkit-box-flex: 0;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    padding-right: 17px;
    border-width: 0;
    border-style: solid;
    border-color: inherit;
}
```

http://plnkr.co/edit/CHn77VE895CXIDGZfmgj?p=preview

**Comment by gyoshev** [comment]
The above snippet is working correctly with the latest `dev` version. Does the problem persist when you update the theme?

**Comment by IanMuirhead** [comment]
@gyoshev It does - must have been stuck between Dev builds.

Closing issue - thanks for the response.

**Comment by gyoshev** [comment]
The patch has been released officially in `2.39.0`
